addons/l10n_ie/data/template/account.tax.group-ie.csv

```diff
@@ -1,2 +1,7 @@
-"id","country_id","name"
-"default_tax_group","base.ie","Taxes"
+"id","name","country_id"
+"ie_tax_group_vat_23","VAT 23%","base.ie"
+"ie_tax_group_vat_13_5","VAT 13.5%","base.ie"
+"ie_tax_group_vat_9","VAT 9%","base.ie"
+"ie_tax_group_vat_5","VAT 5%","base.ie"
+"ie_tax_group_vat_4_8","VAT 4.8%","base.ie"
```

Comment:
4.8% and 5% are both still used?  I can not find the 5% mentioned in https://www.citizensinformation.ie/en/money-and-tax/tax/duties-and-vat/value-added-tax/

Reply:
https://www.revenue.ie/en/vat/vat-rates/historical-vat-rates/index.aspx
https://www.revenue.ie/en/vat/vat-rates/what-are-vat-rates/index.aspx
It's not that clear indeed...
It's not present in "Actual rates" but present in history as "starting from 2022".
Either we remove it or I have to change its value to 5.5% and not 5%.
What do you think ?